src/UKHO.ERPFacade.Common/Infrastructure/IEnumerableComparerExtension.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace UKHO.ERPFacade.Common.Infrastructure
+{
+    [ExcludeFromCodeCoverage]
+    public static class IEnumerableComparerExtension
```

Comment:
Thanks Nadeem

I assume this should be EnumerableComparerExtension as the "I" normally prefixes an Interface.

That being the case should this be excluded from Code Coverage?